cloudformation.yaml

```diff
@@ -26,6 +27,26 @@ Parameters:
         Description: Secret see to hash IDs
         Type: String
 
+Globals:
+    Function:
+        Environment:
+            Variables:
+                EMAIL_ORIGIN: !Ref Origin
+                SALT: !Ref Salt
+                BONOBO_URL: !Ref BonoboUrl
+                BONOBO_USERS_TABLE: !Sub bonobo-${Stage}-users
+        CodeUri:
+            Bucket: content-api-dist
+            Key: !Sub ${Stack}/${Stage}/${App}-reminder/${App}.jar
+        MemorySize: 384
+        Runtime: java8
+        Timeout: 100
+        Events:
+            ScheduleLambda:
+                Type: Schedule
+                Properties:
+                    Schedule: cron(12 14 * * * *) # run daily at 14:12
+                    Input: "true"
 Resources:
     LambdaRole:
         Type: AWS::IAM::Role
```

Comment:
We may be able to simplify the policies listed here using built-in AWS policies. For example, we can replace the logging permissions with [`AWSLambdaBasicExecutionRole`](https://console.aws.amazon.com/iam/home?region=eu-west-1#/policies/arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole$serviceLevelSummary).

The full list of built-in roles can be found [here](https://console.aws.amazon.com/iam/home?region=eu-west-1#/policies).